app/controllers/v0/addresses_controller.rb

```diff
@@ -8,8 +8,9 @@ def show
     end
 
     def update
-      # TODO(AJD): raise on missing params, awaiting evss response on req fields
-      response = service.update_address(@current_user, Oj.load(request.body.string))
+      address = EVSS::PCIUAddress::Address.build_address(Oj.load(request.body.string))
```

Comment:
Two questions on this line:
1.  What's the motivation to put the `Address` model under `/lib` instead of `/app/models`? (in this case and generically)
2.  Why `request.body` instead of the `params` hash?